docs/DESIGN_GUIDELINES.md

```diff
@@ -617,7 +617,7 @@ export class MyOption {
   public static COMMON_OPTION_1 = new MyOption('common.option-1');
```

Comment:
Can we change the `MyProps` while we're in the area to read:

```ts
export interface MyProps {
  readonly option: MyOption;
}
```

(Sorry, I can't comment on that line directly, because it was not changed in the PR)

Reply:
Sure, I've updated the PR!